### PR TITLE
Enable CD

### PR DESCRIPTION
**Pull request by jglick**
Builds on top of https://github.com/jenkinsci/plugin-compat-tester/pull/255 after https://github.com/jenkins-infra/repository-permissions-updater/pull/2103. I lack write access to this repo so I would not be able to help troubleshoot issues.

### CODE REVIEW

File: .github/workflows/cd.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      uses: actions/setup-java@v2
+      with:
+        distribution: 'adopt'
+        java-version: 8
+    - name: Release
+      uses: jenkins-infra/jenkins-maven-cd-action@v1.1.0
+      with:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        MAVEN_USERNAME: ${{ secrets.MAVEN_USERNAME }}
+        MAVEN_TOKEN: ${{ secrets.MAVEN_TOKEN }}
```
Comment (jglick) [inline .github/workflows/cd.yaml:59]: Verify that this secret has been saved in **Settings**.

File: plugins-compat-tester-model/pom.xml [hunk 2]
```diff
@@ -53,4 +53,17 @@
...
   	</dependency>
   </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-deploy-plugin</artifactId>
+        <configuration>
+          <skip>true</skip>
```
Comment (jglick) [inline plugins-compat-tester-model/pom.xml:63]: I figure the module structure of this repo is uninteresting to consumers—they only wish to use the CLI artifact.

File: .github/workflows/cd.yaml (hunk 1 above)
Comment (timja) [reply .github/workflows/cd.yaml:59]: image

**Comment by jglick** [comment]
> I lack write access to this repo

Or rather, it seems I have write but lack admin.

**Comment by timja** [comment]
> > I lack write access to this repo
> 
> Or rather, it seems I have write but lack admin.

Changed

File: plugins-compat-tester-model/pom.xml (hunk 2 above)
Comment (jtnord) [reply plugins-compat-tester-model/pom.xml:63]: his is incorrect - anyone that wants to write hooks needs to consume the model.   May aw well just push everything and be done.

**Pull request merged**